app/controllers/reminders/reminder/add-reminder.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    addReminder() {
+      console.log('woot there it is');
```

Comment:
Also whats even happening here? Shouldn't you have logic to add a reminder in this spot?